tests/unit/querybuilder.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { shallowMount } from '@vue/test-utils';
+import QueryBuilder from '@/components/QueryBuilder.vue';
+
+describe( 'QueryBuilder.vue', () => {
+	it( 'renders props.msg when passed', () => {
+		const wrapper = shallowMount( QueryBuilder, {} );
+		expect( wrapper.text() ).toMatch( '' );
+	} );
```

Comment:
Not that this test is super useful either way, but the test message threw me off. How about something like:

```suggestion
	it( 'has a heading', () => {
		const wrapper = shallowMount( QueryBuilder, {} );
		expect( wrapper.find( 'h1' ).text() ).toBe( 'Simple Query Builder' );
	} );
```

Reply:
Yeah, I honestly forgot to fix it *hides*